
As of version 3.9.3, python stopped bundling some autoconf m4 macros and
started relying on the 'autoconf-archive' package being installed and
in standard system paths. Yes, in a minor release update, really.

--- a/configure.ac~	2021-04-03 16:40:23.864080139 +0000
+++ b/configure.ac	2021-04-03 16:41:05.995807771 +0000
@@ -12,6 +12,8 @@
 
 AC_INIT([python],[PYTHON_VERSION],[https://bugs.python.org/])
 
+AC_CONFIG_MACRO_DIRS([/opt/ooce/autoconf-archive/share/aclocal])
+
 m4_ifdef(
     [AX_C_FLOAT_WORDS_BIGENDIAN],
     [],
